styles.css

```diff
@@ -1 +1,93 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.container {
+    display:grid;
+    row-gap: 20px;
+    column-gap: 20px;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 180px auto 50px;
+    grid-template-areas:
+            "header aside aside"
+            "content aside aside"
+            "content aside-small dynamische-aside"
+            "footer footer footer";
+}
+
+.container {
+    display:grid;
+    row-gap: 20px;
+    column-gap: 20px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px;
+    grid-template-areas:
+            "header"
+            "dynamisch-aside"
+            "content"
+            "dynamische-aside"
+            "aside-small"
+            "footer";
+}
+
+.item {
+    background-color: brown;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: white;
```

Comment:
in de uitwerking zie ik hier ook nog een padding bij staan, weet niet of het zonder fout is...?